data/articles/05770.json

```diff
@@ -1,4 +1,6 @@
 {
+    "received": "2014-08-18",
+    "accepted": "2014-08-31",
```

Comment:
Looks here like its been made required for the archive:
https://github.com/elifesciences/api-raml/commit/e5ba4ea1528f3f990494159df19e272254392e65#diff-28d9beff6cba1dc295568da1bd47802bR38